test/server_integration/http/ssl_redirect/config.js

```diff
@@ -38,20 +37,18 @@ export default async function ({ readConfigFile }) {
   const httpConfig = await readConfigFile(require.resolve('../../config'));
   const certificateAuthorities = [readFileSync(CA_CERT_PATH)];
 
-  const redirectPort = httpConfig.get('servers.opensearchDashboards.port') + 1234;
+  const opensearchDashboardsUrl = new URL('', httpConfig.get('servers.opensearchDashboards'));
+  const redirectPort = opensearchDashboardsUrl.port + 1234;
```

Comment:
`opensearchDashboardsUrl.port` is a string
```suggestion
  const redirectPort = parseInt(opensearchDashboardsUrl.port, 10) + 1234;
```